src/CommandLineTests/NServiceBus.Transports.SQS.CommandLine.Tests.csproj

```diff
@@ -10,6 +10,7 @@
   </ItemGroup>
 
   <ItemGroup Label="Required to force the main project's transitive dependencies to be copied">
+    <PackageReference Include="GitHubActionsTestLogger" Version="1.2.0" />
```

Comment:
This should be in the ItemGroup below, not this one.

Reply:
Fixed